src/pages/resource/LinkPublicationPanel.tsx

```diff
@@ -31,18 +30,20 @@ interface LinkPublicationPanelProps {
 }
 
 const LinkPublicationPanel: React.FC<LinkPublicationPanelProps> = ({ expanded, onChange, goToNextTab }) => {
-  const [doiUrl, setDoiUrl] = useState('');
   const { t } = useTranslation();
+  const [doiUrl, setDoiUrl] = useState('');
+  const [doiTitle, setDoiTitle] = useState('');
```

Comment:
hva med å slå sammen til `const [doi, setDoi] = useState(null);`, som kan bestå av et DOI-objekt?

Reply:
Det er vel forhåpentligvis et "resource"-objekt vi skal få tilbake fra tjenesten og så bruke for å populere formen, men siden det er så uklart, tok jeg kun ut tittelen for å gjenskape GUI-tegningene.

Reply:
tommelfinger-regelen er vel at når man har 3 eller flere useStates på samme type, så burde man slå det sammen, men synes det er greit at det er to stk